docker/justfile

```diff
@@ -33,6 +33,8 @@ check-js:
 check-py: build
     docker compose run --rm dev /app/docker/entrypoints/check.sh
 
+check-migrations *args="":
```

Comment:
should this call the `build` recipe like `check-py` does? The errors I were seeing were due to not having run `build` manually in advance.